### PR TITLE
feat(frontend): Initial Designs of Quiz Components and Pages

### PR DESCRIPTION
**Pull request by Elifnurdeniz**
[Issue](https://github.com/bounswe/bounswe2024group6/issues/454)

### CODE REVIEW

File: frontend/src/pages/quizzes.tsx [hunk 1]
```diff
@@ -1,78 +1,42 @@
...
+        tags={quiz.tags}
+      />
+      <QuizCard
+        key={quiz.id}
+        id={quiz.id}
+        username={quiz.author}
+        title={quiz.title}
+        content={quiz.content}
+        timePassed={quiz.timestamp}
+        likeCount="1000"
```
Comment (alitariksahin) [inline frontend/src/pages/quizzes.tsx:35]: pass a number, not a string :)
image

**Review by alitariksahin — COMMENTED** [review]
well done!

File: frontend/src/pages/quizzes.tsx (hunk 1 above)
Comment (Elifnurdeniz) [reply frontend/src/pages/quizzes.tsx]: It's resolved

**Review by alitariksahin — APPROVED** [review]
wow

**Pull request merged**